src/style.css

```diff
@@ -0,0 +1,119 @@
+@import url(./reset.css);
+
+body {
+  background-color: var(--grey-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card {
+  box-sizing: border-box;
```

Comment:
move this property to `*` selector to apply for every element
remove it in all other places